src/gorilla.c

```diff
@@ -256,7 +256,7 @@ static inline binary_t readBits(const binary_t *bins,
     }
 }
 
-static bool isSpaceAvailable(CompressedChunk *chunk, u_int8_t size) {
+static inline bool isSpaceAvailable(CompressedChunk *chunk, u_int8_t size) {
```

Comment:
@OfirMos using really_inline here?

Reply:
I think that if the compiler really don't want it to be inlined it has a good reason for this, do you have any proof that it's recommended to use really_inline?

Reply:
@filipecosta90 I think that if the compiler really against inline the function we shouldn't enforce it. Have you find a place which recommends using it?